examples/Responsive.js

```diff
@@ -10,6 +21,7 @@ export default class Responsive extends Component {
       slidesToShow: 4,
       slidesToScroll: 4,
       initialSlide: 0,
+      unslick: UnSlick,
```

Comment:
Do we really need this feature added to the react-slick.
It can be easily implemented by unmounting Slider and mounting UnSlick component